rpc/client/rpc_test.go

```diff
@@ -414,14 +415,25 @@ func TestTx(t *testing.T) {
 }
 
 func TestTxSearch(t *testing.T) {
-	// first we broadcast a tx
 	c := getHTTPClient()
-	_, _, tx := MakeTxKV()
-	bres, err := c.BroadcastTxCommit(tx)
-	require.Nil(t, err)
 
-	txHeight := bres.Height
-	txHash := bres.Hash
+	// first we broadcast a few txs
+	var tx []byte
```

Comment:
small nit:

```
var (
)
```

Reply:
This should really be enforced by a linter.